tests/unit/src/test/scala/tests/SimpleFileWatcherSuite.scala

```diff
@@ -0,0 +1,125 @@
+package tests
```

Comment:
I don't see a `BUILD` or `BUILD.bazel` file in this directory. Is there anything more that needs to be done to get this to run, or is merely creating this file enough?

Reply:
Good question. I was able to run it with sbt, but I think there was a warning about adding it to a list or something. I'll do that.

Reply:
Ok, I added it to a test group. I think that just decides which thread to run it on when the ci runs tests, but sbt definitely found it however that works. :man_shrugging: